src/Http/Controllers/ButtonController.php

```diff
@@ -22,7 +22,7 @@ public function handle(NovaRequest $request)
         $event = $request->get('event');
         $resource = $request->findModelQuery()->firstOrFail();
 
-        event(new $event($resource, $request->get('buttonKey')));
+        event(new $event($resource, $request->route('buttonKey')));
```

Comment:
this is actually not a type, this does what it needs to do get the buttonKey attribute from the route.

Reply:
I just test it and I get `null` from `$request->get('buttonKey')`